scripts/zones/Cape_Teriggan/Zone.lua

```diff
@@ -66,7 +62,7 @@ zoneObject.onZoneWeatherChange = function(weather)
 
     if
         not kreutzet:isSpawned() and
-        os.time() > kreutzet:getLocalVar("cooldown") and
+        os.time() > GetServerVariable("[SPAWN]17093082") and
```

Comment:
Is this unavoidable in the NM persistence setup? Would prefer to not use a mob's ID permanently like this.

Reply:
this is what NM persistence does. it sets a server variable based from their ID

Reply:
However I don't believe it's unavoidable as mob variables are wiped upon server restart